codalab/lib/beam/MultiReaderFileStream.py

```diff
@@ -10,11 +10,16 @@ class MultiReaderFileStream(BytesIO):
     """
     NUM_READERS = 2
 
+    # MAX memory usage <= MAX_BUF_SIZE + max(num_bytes called in read)
```

Comment:
What does this comment mean? Can you add a description of what MAX_BUF_SIZE is used for?